torch/csrc/jit/passes/quantization.cpp

```diff
@@ -243,28 +221,56 @@ Node* InsertObserversHelper::insertObserverFor(
   return call;
 }
 
-void fillQConfigMap(
-    const script::Module& module,
-    const QConfigDict& qconfig_dict,
-    ModuleQConfigMap& map,
-    const std::string& key = "",
-    const c10::optional<QConfig>& parent_qconfig = c10::nullopt) {
-  c10::optional<QConfig> qconfig;
-  if (qconfig_dict.find(key) != qconfig_dict.end()) {
-    qconfig = qconfig_dict.at(key);
-  } else {
-    qconfig = parent_qconfig;
+void InsertObserversHelper::findValuesInPattern(
```

Comment:
The name of the function doesn't reflect its details. It takes many assumption and does a very specific thing - looks for a value in the pattern with a specific name "output" and adds this to a map. We need to call out these assumptions in the comments and better reflect it in the name. Also, I think it might be worth renaming "output" to "intermediate" or something like that.

Reply:
sounds good